nvflare/private/fed/app/server/server_train.py

```diff
@@ -82,6 +89,11 @@ def main(args):
             logger.critical("loglevel critical enabled")
         conf.configure()
 
+        decomposer_module = ConfigService.get_str_var(
```

Comment:
We shouldn't need to lode any custom decomposers this since this is SP.

Reply:
This does not hurt. We always do this fobs initialize, https://github.com/yhwen/NVFlare/blob/fobs_auto_register/nvflare/private/fed/app/server/server_train.py#L73

Reply:
This could hurt if there is bug in custom decomposers.
As we discussed before, SP and CP are NVFLARE's code only and shouldn't have anything to do with user code.

Reply:
Removed the custom decomposer register for SP.